app/models/miq_alert_set.rb

```diff
@@ -12,7 +12,7 @@ def self.assigned_to_target(target, options = {})
   end
 
   def notes
-    set_data.kind_of?(Hash) && set_data.key?(:notes) ? set_data[:notes] : nil
+    set_data.kind_of?(Hash) && set_data[:notes]
```

Comment:
Not sure, but this might be intentional if set_data has a default proc.  In the case of a default proc checking the key is the only way to know if something actually exists in the hash or not.

Reply:
good enough for me to revert. I don't want to overly think this one